content/news/2023-05-25-maplibre-gl-js-v3/index.md

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
See Open Graph and Twitter Cards: https://gohugo.io/templates/internal/.

I would suggest to add "description" and "images" in Front Matter for this post. "images" is missing under "params": https://github.com/maplibre/maplibre.github.io/blob/main/config.toml#L24.

Reply:
i.e. images: ['gl-js-v3.png'] if https://github.com/maplibre/maplibre.github.io/pull/171#issuecomment-1559212865 is to be used as a preview

Reply:
As in https://github.com/maplibre/maplibre.github.io/blob/main/layouts/partials/head.html#L23 add 

images = "/img/share-image.png"

to https://github.com/maplibre/maplibre.github.io/blob/main/config.toml#L24

Reply:
share-image.png currently is https://github.com/maplibre/maplibre.github.io/blob/main/static/img/share-image.png